Shared/Extensions/UIImageExtensions.swift

```diff
@@ -61,4 +61,22 @@ extension UIImage {
     public static func templateImageNamed(_ name: String) -> UIImage? {
         return UIImage(named: name)?.withRenderingMode(.alwaysTemplate)
     }
+
+    // TESTING ONLY: not for use in release/production code.
```

Comment:
Can we do better than a comment block? e.g. 

```swift
@available(*, deprecated, message: "use only in testing code")
```

or put in the test case file instead?

Reply:
That `available` usage is great, I'll go with that.
Indeed it could go in the file it is used in (I removed the usage of this type of comparison in Storage tests), but I want this very discoverable, because someone could end up doing this incorrectly and costing themselves cycles -like me :).